lib/tasks_private/spec_helper.rake

```diff
@@ -434,6 +434,20 @@ class PopulateTower
     last_update = wait_for_project_update(jobless_project)
     @conn.delete(last_update['url'])
 
+    # Create and remove project - record an collect an ID of missing entity
+    uri = '/api/v1/projects/'
+    uri = '/api/v1/projects/'
+    data = {
+      :name         => 'nonexistent_repo',
+      :scm_url      => 'https://github.com/jameswnl/ansible-examples',
+      :scm_type     => 'git',
+      :credential   => scm_credential['id'],
+      :organization => organization['id']
+    }
+    nonexistent_project = create_obj(uri, data)
+    @tower_data[:items][data[:name]] = { :id => nonexistent_project['id'] }
+    del_obj(uri, data[:name])
```

Comment:
Just remember, that `del_obj` does some safety sleep. This might be necessary when recreating object that are already present in Tower. It is however useless here, when we just want to delete an object without any intention of creating its replacement.

https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/93/commits/7f4bdfeb147d0991b05db858cb5d5a7861f05fdb#diff-bb7272be93a294c53ae6433eb024074fR108

Reply:
What's the issue with this? The rake job is run only in case of regenerating VCRs. It doesn't slow down the test run or anything...

Reply:
You just wait a *long* time for nothing before going to the next step – populating item counts.